docs/integrations/azure/org.md

```diff
@@ -125,6 +125,17 @@ microsoftGraphOrg:
 In addition to these groups, one additional group will be created for your organization.
 All imported groups will be a child of this group.
 
+By default the provider will get groups using the msgraph `/group` endpoint, but it is possible to use different endpoints by setting the `path` configuration. All the endpoint containing `/microsoft.graph.group` will return the right type of group object.
```

Comment:
Maybe, a visualization like the one in your PR description could be helpful here as well. Different shapes for `group` and `user` might be an idea as well.

Reply:
Hey, sry I totally missed the comment, I will work on it.  thanks for reviewing!!

Reply:
@pjungermann I moved the docs to its own section  this is how it looks now

![image](https://github.com/user-attachments/assets/2e2dff1a-d1ce-4194-84a6-c7c31dcded76)